keras_nlp/models/opt/opt_causal_lm.py

```diff
@@ -449,3 +449,39 @@ def preprocess(x):
         if outputs.dtype == tf.string:
             return tensor_to_string_list(outputs)
         return outputs.numpy()
+
+    @classmethod
```

Comment:
Static method?

Reply:
Cannot, note the use of `cls` below. If you have a different vision in mind lmk!